src/week08/lecture/staking.py

```diff
@@ -0,0 +1,43 @@
+from opshin.prelude import *
+
+
+def equal(a: Union[StakingHash, StakingPtr], b: Union[StakingHash, StakingPtr]) -> bool:
```

Comment:
There should be no need to write this function. You can just cast both a and b to Union and they should be comparable via `==`